COMMPLAN.md

```diff
@@ -0,0 +1,204 @@
+
+Use our blog as a way of publicizing new software package, new software
+versions, solutions to hard problems, or any general accomplishment.
+Use our blog to make *general public announcements* about things like
+tutorials or new services.  Use our blog to publicize Xdev Team work and
+achievements.
+
+#### What should we *not* use it for?
+
+Don't use the blog for rants or making complaints about anything.  The
+blog should only be used for *constructive* communication.
+
+### Twitter (@NCARXdev)
+
+#### Why we should use it?
+
+Twitter is a commonly used platform by many people in the Tech Industry,
+especially software engineers and scientists.  For example, a very large
+fraction of the Pangeo community regularly tweet.  Hence, Twitter makes
+an excellent platform for getting the attention of the general public
+(or the fraction of the general public interested in NCAR's Python
+work).
+
+#### What should we use it for?
+
+The Xdev Twitter account should be for making *announcements* to the
+general public, such as making an announcement of a new blog post.
+In general, Twitter *could* be used for any announcement, but because
+of the limitations of the platform (i.e., limiting the length of the
+tweet) it is best to tie the tweet to *another* communication platform,
+such as a blog post.
+
+#### What should we *not* use it for?
+
+The Xdev Twitter account should *not* be used for personal use.  Currently,
+only Kevin has the "keys" to the Twitter account.  If you want to make a
+broad announcement, contact Kevin via Zulip.
+
+### Email
+
+#### Why should we use it?
+
+Not sure.  I'm open to discussion on this, but I am beginning to feel like
+email (especially the `xdev@ucar.edu` address) should not be used by any
+of us, unless people explicitly reach out to us.  That is, I think email
+should be a way for people *outside* of NCAR to directly communicate with
+*us*.
+
+At NCAR, email is used substantially for inter-NCAR communication or for
+communication with specific extra-NCAR entities (like the NSF).  The amount
+of email that any one of us can easily receive is enough that it is easy to
+get lost, and so I recommend *not* using email for Team communication.
+
+#### What should we use it for?
+
+Just to reply to other people who use the `xdev@ucar.edu` address.
+
+#### What should we *not* use it for?
+
+Pretty much everything.
+
+### Other Channels
+
+There are many other communication channels for other (albeit similar)
+communities, even ones that we might consider ourselves members.  The
+most notable case is Pangeo Community, which has its own communication
+channels, as described in the [Pangeo Guide](PANGEO.md).  If directly
+interacting with an external community, such as Pangeo, use the communication
+channels *used by that community.*  Note that this could me, regretfully,
+re-posting or double-posting announcements or questions in multiple
+channels because of different community expectations.
+
+## Summary Chart
```

Comment:
No recommended changes for this part, I just want to 👍 this entire section -- it's a great summary

Reply:
Thanks, @mnlevy1981!